pact/Pact/Core/IR/Term.hs

```diff
@@ -348,6 +351,9 @@ instance (Pretty name, Pretty builtin, Pretty ty) => Pretty (Term name ty builti
       parens ("try" <+> pretty te <+> pretty te')
     ObjectLit n _ ->
       braces (hsep $ punctuate "," $ fmap (\(f, t) -> pretty f <> ":" <> pretty t) n)
+    InlineValue{} ->
```

Comment:
Even though this variant is only used in legacy terms, is there a chance we could encounter one why pretty-printing a legacy value? If so' should we give this variant a normal `pretty` representation?

Reply:
Yes, I agree I think we should